src/features/projects/components/ProjectSnippet.tsx

```diff
@@ -41,26 +41,28 @@ export default function ProjectSnippet({
     progressPercentage = 100;
   }
 
-  const [open, setOpen] = React.useState(false);
-  const handleClose = () => {
-    setOpen(false);
-  };
+  const { user } = React.useContext(UserPropsContext);
+
+  const currency = getStoredCurrency();
+  const country = localStorage.getItem('countryCode');
+  const language = localStorage.getItem('language');
+  let directGift = localStorage.getItem('directGift');
+  if (directGift) {
+    directGift = JSON.parse(directGift);
+  }
+
+  const getSourceUrl = React.useCallback((): string => {
+    var sourceUrl = `${process.env.NEXT_PUBLIC_DONATION_URL}/?to=${project.slug}&returnToUrl=${window.location.href}&country=${country}&currency=${currency}&locale=${language}${user ? '&autoLogin=true' : ''}&tenant=${process.env.TENANTID}${directGift ? '&s=' + directGift.id : ''}`;
```

Comment:
`NEXT_PUBLIC_DONATION_URL=https://donate.plant-for-the-planet.org` should be added to our .env files.

Reply:
added it